google/cloud/testing_util/validate_metadata.cc

```diff
@@ -248,9 +248,10 @@ RoutingHeaders ExtractRoutingHeaders(
  *   cannot reuse @p context for other RPCs or other calls to this function.
  */
 void SetServerMetadata(grpc::ClientContext& context,
-                       RpcMetadata const& server_metadata) {
+                       RpcMetadata const& server_metadata,
+                       bool is_initial_metadata_ready) {
```

Comment:
We should not make these changes to `testing_util`.

Reply:
Done